docker/package/wizard_structure.py

```diff
@@ -302,6 +413,81 @@ def pprint_options(self):
                 else:
                     print(str_format.format(i, o, description))
                 i += 1
+        elif not self.options and self.default is not None:
+            print("Default:", self.default)
```

Comment:
Now wizard prints some weird lines:
```
How do you want to import the voter key?
1. ledger (default)            From a ledger
2. secret-key                  Either the unencrypted or password-encrypted secret key for
                               your address
3. remote                      Remote key governed by a signer running on a different
                               machine
> 2
Provide either the unencrypted or password-encrypted secret key for your address.
Default: 1
>
```

Reply:
Well, AFAIU that step shouldn't have a default anyway